exemplar/web/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "exemplar",
+  "version": "0.1.0",
```

Comment:
for me to follow up – are all of these required? Can we omit the version etc.?

Reply:
You can specifiy a minimum version but that's just asking for trouble.